packages/webservice/app/src/main/resources/db/changelog/changelog.xml

```diff
@@ -161,6 +161,8 @@
                 ADD isDisabled boolean not null default false;
             ALTER TABLE DeliveryRule 
                 ADD failureCount int not null default 0;
+            ALTER TABLE DeliveryRule    
+                ADD name varchar(255) not null;
         </sql>
     </changeSet>
```

Comment:
I think we'll have to add a serializer so that the app automatically updates the delivery rule name to its default because it will start out null since we don't have a default in the sql statement.

Otherwise we can have  a default like `"My Delivery Rule"` and keep the not null constraints

```suggestion
        </sql>
    </changeSet>
    <changeSet author="admin" id="5">
        <sql>
            ALTER TABLE DeliveryRule
                ADD name varchar(255);
        </sql>
    </changeSet>
```